br/pkg/trace/tracing_test.go

```diff
@@ -10,37 +10,29 @@ import (
 	"time"
 
 	"github.com/opentracing/opentracing-go"
-	. "github.com/pingcap/check"
+	"github.com/stretchr/testify/require"
 )
 
-type testTracingSuite struct{}
-
-var _ = Suite(&testTracingSuite{})
-
-func TestT(t *testing.T) {
-	TestingT(t)
-}
-
 func jobA(ctx context.Context) {
 	if span := opentracing.SpanFromContext(ctx); span != nil && span.Tracer() != nil {
 		span1 := span.Tracer().StartSpan("jobA", opentracing.ChildOf(span.Context()))
 		defer span1.Finish()
 		ctx = opentracing.ContextWithSpan(ctx, span1)
 	}
 	jobB(ctx)
-	time.Sleep(time.Second)
+	time.Sleep(10 * time.Millisecond)
 }
 
 func jobB(ctx context.Context) {
 	if span := opentracing.SpanFromContext(ctx); span != nil && span.Tracer() != nil {
 		span1 := span.Tracer().StartSpan("jobB", opentracing.ChildOf(span.Context()))
 		defer span1.Finish()
 	}
-	time.Sleep(time.Second)
+	time.Sleep(10 * time.Millisecond)
 }
 
-func (t *testTracingSuite) TestSpan(c *C) {
-	filename := path.Join(c.MkDir(), "br.trace")
+func TestSpan(t *testing.T) {
```

Comment:
Please check:

* if the test should be run in serial, rename the file as `tracing_serial_test.go`
* otherwise, add `t.Parallel`

Reply:
I think it should run in serial, I will fix it.

Reply:
This test will change global var `getTraceFileName`. I think it should be run in serial.